src/components/measurements/measurementsD3.js

```diff
@@ -138,17 +132,11 @@ const drawMeanAndStandardDeviation = (values, d3ParentNode, containerClass, colo
       .attr("y1", yValue)
       .attr("y2", yValue)
       .attr("stroke-width", layout.standardDeviationStroke)
-      .attr("stroke", color)
-      .on("mouseover", (d) => {
-        // Get mouse position for HoverPanel
-        const { clientX, clientY } = d3event;
-        handleHover(d, "mean", clientX, clientY);
-      })
-      .on("mouseout", () => handleHover(null));
+      .attr("stroke", color);
```

Comment:
> With the color-by attribute titles, the handleHover function will be dependent on color-by changes, which we do _not_ want to force SVG redrawing.

Could you elaborate on this a bit more? Changes in color-by will necessitate some degree of SVG redrawing -- at least the lines (mean+sd) / circles (raw). Does this change stop the `drawMeasurementsSVG` re-rendering, as this does not actually set the colouring [1], whereas (e.g.) `drawMeansForColorBy` will still rerun as required (e.g. when color-by changes)?

[1] If I understand the code, it does run `drawMeanAndStandardDeviation` which sets the color to `layout.overallMeanColor`, but this just adds the elements to the DOM in a non-visible fashion. We need a second call to `drawMeanAndStandardDeviation` to show/color them. Similar for the raw dots.

Reply:
> Does this change stop the drawMeasurementsSVG re-rendering, as this does not actually set the colouring [1], whereas (e.g.) drawMeansForColorBy will still rerun as required (e.g. when color-by changes)?

Yes, this is exactly why I made this change. Previously, `handleHover` was directly tied to the redrawing of the entire SVG. Now, it its only tied to the coloring changes so we don't have to worry about unnecessary SVG redrawings.